docs/shared-content-source/docs/modules/develop/pages/use-spark-streamlets.adoc

```diff
@@ -63,6 +63,16 @@ It contains the following context data and contracts:
 * The name used in the blueprint for the specific instance of this streamlet being run.
 * A function `checkpointDir` which returns a directory on a persistent storage where checkpoints can be safely kept, making them available across restarts.
 
+== Controlling Streaming Query execution
+
+Spark Structured streaming provides mechanism for controlling timing of streaming data processing in the form of
+https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html#triggers[triggers].
+The trigger settings of a streaming query defines whether the query is going to be executed as micro-batch query with a fixed
```

Comment:
"define" instead of "defines" because it refers to settings. as "a" micro-batch query ...